airflow/_vendor/README.md

```diff
@@ -0,0 +1,37 @@
+# Vendor package
+
+## What vendored packages are for
+
+The `airlfow._vendor` package is foreseen for vendoring in packages, that we have to modify ourselves
+because authors of the packages do not have time to modify them themselves. This is often temporary
+and once the packages implement fixes that we need, and then we remove the packages from
+the _vendor package.
+
+All Vendored libraries must follow these rules:
+
+1. Vendored libraries must be pure Python--no compiling.
```

Comment:
Can you elaborate this please

Reply:
This means that we should not vendor-in code that woudl requrire us to produce platform/architecture specific packaes (such as numpy etc.). Otherwise it will require from us to build multiple binary wheels for `airflow` core - where wa only need one pure-python only.